core-web/libs/sdk/react/src/lib/components/Container/Container.tsx

```diff
@@ -44,33 +44,27 @@ export function Container({ containerRef }: ContainerProps) {
     const { identifier, uuid } = containerRef;
 
     // Get the containers from the global context
-    const { containers, page, viewAs, components, isInsideEditor } =
-        useContext<PageProviderContext | null>(PageContext) as PageProviderContext;
+    const { containers, components, isInsideEditor } = useContext<PageProviderContext | null>(
+        PageContext
+    ) as PageProviderContext;
 
-    const { acceptTypes, contentlets, maxContentlets, pageContainers, path } = getContainersData(
+    const { acceptTypes, contentlets, maxContentlets, path } = getContainersData(
         containers,
         containerRef
     );
 
     const updatedContentlets =
         contentlets.length === 0 && isInsideEditor ? [FAKE_CONTENLET] : contentlets;
 
-    const contentletsId = updatedContentlets.map((contentlet) => contentlet.identifier);
-
     const container = {
         acceptTypes,
-        contentletsId,
         identifier: path ?? identifier,
         maxContentlets,
         uuid
     };
 
     const containerPayload = {
-        container,
-        language_id: viewAs.language.id,
-        pageContainers,
-        pageId: page.identifier,
-        personaTag: viewAs.persona?.keyTag
+        container
     };
```

Comment:
Wo we neeed the `container` property here now?

Reply:
Yes, we need it for the Dropzone to work, otherwise we would need to iterate through all the Row -> Column -> Container -> Contentlet tree to insert the data